org.eclipse.jdt.ui/ui refactoring/org/eclipse/jdt/internal/ui/refactoring/refactoringui.properties

```diff
@@ -109,11 +109,13 @@ SelfEncapsulateFieldInputPage_public=&public
 SelfEncapsulateFieldInputPage_default=p&ackage
 SelfEncapsulateFieldInputPage_protected=pro&tected
 SelfEncapsulateFieldInputPage_private=pri&vate
+SelfEncapsulateFieldInputPage_same=&same as field
 SelfEncapsulateFieldInputPage_generateJavadocComment=Generate method &comments
 SelfEncapsulateFieldInputPage_useexistingsetter_label=(using existing setter)
 SelfEncapsulateFieldInputPage_useexistinggetter_label=(using existing getter)
 SelfEncapsulateFieldInputPage_usenewgetter_label=(new setter created)
 SelfEncapsulateFieldInputPage_usenewsetter_label=(new getter created)
+SelfEncapsulateFieldInputPage_access_warning=Some getters/setters are less visible than their field
```

Comment:
Change this to be ... less visible than their original field

since we are resetting the field to be private, it will make more sense to the end-user